openedx.yaml

```diff
@@ -0,0 +1,8 @@
+# This file describes this Open edX repo, as described in OEP-2:
+# http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html#specification
+
+nick: grbk
+oeps: {}
```

Comment:
Since the `oeps` key is optional, I think you can safely eliminate it.

Reply:
huh it does say optional, yet the other projects I'm looking at have an empty object.

Reply:
Hmm, I guess it doesn't really matter.  You can leave as is.